### DISCUSSION TITLE
Custom data/Log file location for Add-DbaAgDatabase

### DISCUSSION THREAD

**Discussion by MuruArumugam** [dataplat/dbatools#7076] [General]
In Add-DbaAgDatabase function Restore-DBADatabase does not use DestinationDataDirectory or DestinationLogDirectory. Any reason?
If I want to add those 2 parameters and create my own function, shred functions like Test-Bound, Write-Message etc don't work(tried to add it on  dbatool.psd1 and Import-Module, didn't help), 

Thanks in advance

Muru

**Comment by wsmelton** [comment]
For custom file locations recommended path is to use the raw backup/restore functions and run add the database to the AG using the AG function.
